packages/react-magma-dom/src/components/AlertBase/index.tsx

```diff
@@ -386,6 +386,19 @@ const AlertSpan = styled.span`
   white-space: pre-line;
 `;
 
+function getAriaLabelIcon(variant: string): string {
+  switch (variant) {
+    case 'success':
+      return 'success icon';
+    case 'warning':
+      return 'warning icon';
+    case 'danger':
+      return 'danger icon';
+    default:
+      return 'info icon';
```

Comment:
Can we add this to our i18n file? Just to be super thorough, in case anybody uses this component with a different language